.github/workflows/pages.yml

```diff
@@ -2,7 +2,7 @@
 name: "Pages"
 
 on:
-  pull_request:
+  push:
```

Comment:
I changed that on purpose. Most changes here happen via pull_requests and not pushes. Especially not in the "main" branch.

Reply:
This won't work. As it only needs to build pages on those branches and not every pull request pointing to those branches. See GitHub actions before last commit.

Reply:
Got it now. Thx for your explanation!